### DISCUSSION TITLE
dh key too small  issue on Postal V1 Version

### DISCUSSION THREAD

**Discussion by rccess** [postalserver/postal#2128] [Help with using Postal]
Hello team,

i have installed Letsencrypt SSL properly 

but having issue 
----------------------------------
SSL_connect returned=1 errno=0 state=error: dh key too small

Support Ref: U4EY3OOT
----------------------------------
configuration 

<img width="500" alt="image" src="https://user-images.githubusercontent.com/32220165/186977375-ce8fd75f-dd0d-449c-98ac-77e886788a8e.png">

Already followed past issues [#1912,#1641,#878,#943,#990] but no luck 

Please do advise 

<img width="520" alt="image" src="https://user-images.githubusercontent.com/32220165/186977990-eac74a50-22fd-40c2-a171-299c1cf9c1d7.png">

**Comment by willpower232** [comment]
Either your Postal server or wherever you're trying to send mail from/to has a wildly different version of OpenSSL with different encryption handling available.

If lowering your Postal servers OpenSSL settings as per the previous issues isn't working then either you've missed something or you'll need to work with whatever you're sending mail from/to in order to investigate this issue further.
